File: november_and_december_2022/SmallerThanCurrentNumber.py
class Solution:
    def smallerNumbersThanCurrent(self, nums: List[int]) -> List[int]:
        returnList = []
        for num in nums:
            counter = 0
            for otherNums in nums:
                if num > otherNums:
                    counter += 1
            returnList.append(counter)
        return returnList
        

File: november_and_december_2022/SmallerThanCurrentNumber.java
class Solution {
    public int[] smallerNumbersThanCurrent(int[] nums) {
       int[] result = new int[nums.length];
        int index = 0;
        for (int num : nums){
        int counter = 0;
            for (int otherNum : nums){
                if (otherNum < num){
                    counter += 1;
                }
            }
            result[index] = counter;
            index += 1;
            
        }
        return result;
    }
}
